app/components/in-view.js

```diff
@@ -0,0 +1,34 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
```

Comment:
I haven't checked all browsers, because my main development environment is a chromebook. How big is your mopidy library? Try entering a search query, like `a`, which returns 1,000s of results and you will see what I mean.